### ISSUE TITLE
Code checker does not allow tabs after `//`

### ISSUE THREAD

**Issue by JanTvrdik** [nette/code-checker#15]
The following code is considered invalid because of tabulators after `//`. Feature or bug?

```
//          '5.3 Other illegal code positions' => array(
//              '5.3.1  U+FFFE = ef bf be' => array(
//                  "\xEF\xBF\xBE",
//                  "",
//              ),
//              '5.3.2  U+FFFF = ef bf bf' => array(
//                  "\xEF\xBF\xBF",
//                  "",
//              ),
//          ),
```

**Comment by dg** [comment]
fixed 094c43020ea12974dc96b52d8c3724983b1c3df4

**Comment by JanTvrdik** [comment]
Thank you!
